daemon/execdriver/driver.go

```diff
@@ -93,6 +93,11 @@ type Pid struct {
 	HostPid bool `json:"host_pid"`
 }
 
+// UTS settings of the container
+type Uts struct {
+	HostUts bool `json:"host_uts"`
```

Comment:
In the future we could support --uts=container:id, but I don't know what the use case is for that.  This approach is compatible with that code change in the future as we followed the approach as Ipc with supports both --ipc=host and --ipc=container:id